document/src/main/java/com/ritense/document/service/SearchFieldService.java

```diff
@@ -52,4 +52,9 @@ public void createSearchConfiguration(String documentDefinitionName, List<Search
             searchFieldRepository.saveAll(searchFields);
         }
     }
+
+    public void deleteSearchFields(String documentDefinitionName, String key) {
```

Comment:
This says field*s* (plural), but shouldn't it be singular?

Reply:
you're right. will fix